components/molecule/select/src/index.scss

```diff
@@ -40,6 +40,11 @@ $bgc-atom-input-disabled: $c-gray-lightest !default;
         }
       }
 
+      #{$class-select-atom-input-input}:disabled {
+        opacity: 1;
```

Comment:
This won't work if you move this properties into the line 39? It seems it's applying to the same selector...

Reply:
I think whatever there is on L39 applies to #{$class-select-atom-input-input} within an explicitly disabled molecule select. In our case we want to make sure it's applying to the disabled input element of the atom input, whether or not the molecule select containing it is disabled.

Reply:
ok, it makes sense